x-pack/plugins/observability_solution/apm/public/components/routing/templates/apm_main_template/index.tsx

```diff
@@ -168,47 +162,45 @@ export function ApmMainTemplate({
     </EuiFlexGroup>
   );
 
-  const pageTemplate = showCustomEmptyState ? (
-    <CustomNoDataTemplate isPageDataLoaded={isLoading === false} noDataConfig={noDataConfig} />
-  ) : (
-    <ObservabilityPageTemplate
-      noDataConfig={
-        shouldBypassNoDataScreen ||
-        serviceInventoryViewLocalStorageSetting === ServiceInventoryView.entity
-          ? undefined
-          : noDataConfig
-      }
-      isPageDataLoaded={isLoading === false}
-      pageHeader={{
-        rightSideItems,
-        ...pageHeader,
-        pageTitle: pageHeaderTitle,
-        children: (
-          <EuiFlexGroup direction="column">
-            {isEntityCentricExperienceSettingEnabled &&
-            showEnablementCallout &&
-            selectedNavButton === 'allServices' ? (
-              <EntityEnablement
-                label={i18n.translate('xpack.apm.eemEnablement.tryItButton.', {
-                  defaultMessage: 'Try our new experience!',
-                })}
-                tooltip={i18n.translate('xpack.apm.entityEnablement.content', {
-                  defaultMessage:
-                    'Our new experience combines both APM-instrumented services with services detected from logs in a single service inventory.',
-                })}
-              />
-            ) : null}
-            {showServiceGroupsNav && selectedNavButton && (
-              <ServiceGroupsButtonGroup selectedNavButton={selectedNavButton} />
-            )}
-          </EuiFlexGroup>
-        ),
-      }}
-      {...pageTemplateProps}
-    >
-      {children}
-    </ObservabilityPageTemplate>
+  const [dismissedEntitiesInventoryCallout, setdismissedEntitiesInventoryCallout] = useLocalStorage(
+    `apm.dismissedEntitiesInventoryCallout`,
+    false
   );
 
-  return <EnvironmentsContextProvider>{pageTemplate}</EnvironmentsContextProvider>;
+  const showEntitiesInventoryCallout =
+    (!dismissedEntitiesInventoryCallout &&
+      isEntityCentricExperienceSettingEnabled &&
+      selectedNavButton === 'allServices') ||
+    selectedNavButton === 'serviceGroups';
```

Comment:
This looks over complicated. We basically want to show the callout no matter which page the user is. 

```suggestion
  const showEntitiesInventoryCallout = !dismissedEntitiesInventoryCallout && isEntityCentricExperienceSettingEnabled
```

Reply:
Sorry I've just realized this should be:

````
const showEntitiesInventoryCallout =
    !dismissedEntitiesInventoryCallout &&
    isEntityCentricExperienceSettingEnabled &&
    selectedNavButton !== undefined;
````

We can't remove the `selectedNavButton` check because this is a template being used in the service detail view too and we don't want the callout there:

![Screenshot 2024-09-27 at 10 49 06](https://github.com/user-attachments/assets/2f933ae7-57ad-4656-9f7e-5cd92526b55d)